run_tests.sh

```diff
@@ -2,15 +2,31 @@
 set -ex
 
 dir=$(pwd)
-# list of all modules to test
-modules=". /dex/testing/loadbot /client/cmd/dexc-desktop"
 
 GV=$(go version | sed "s/^.*go\([0-9.]*\).*/\1/")
 echo "Go version: $GV"
 
 # Ensure html templates pass localization.
 go generate -x ./client/webserver/site # no -write
 
+# Ensure cache busters are updated
+set +x
+source ./client/webserver/site/cache_utilities.bash
+CSS_HASH=`hashcsssrc`
+CSS_BUSTER=`csssrcbuster`
+JS_HASH=`hashjssrc`
+JS_BUSTER=`jssrcbuster`
+if [ "${CSS_HASH}" != "${CSS_BUSTER}" ] || [ "${JS_HASH}" != "${JS_BUSTER}" ]; then
+	setcssbuster
```

Comment:
If it gets in here it will have to first do `npm ci && npm run build` before running the `set` helpers